src/shared/components/mutationTable/column/DiscreteCNAColumnFormatter.tsx

```diff
@@ -18,15 +18,186 @@ export default class DiscreteCNAColumnFormatter {
         "-2": "deepdel"
     };
 
-    public static renderFunction(data: Mutation[], molecularProfileIdToMolecularProfile: {[molecularProfileId:string]:MolecularProfile}, cache:DiscreteCNACache) {
+    private static getFacetsCNAData(data:Mutation[], sampleIdToClinicalDataMap:{[sampleId:string]:ClinicalData[]}|undefined) {
+        let sampleId:string = data[0].sampleId;
+        let tcn = data[0].totalCopyNumber;
+        let lcn = data[0].minorCopyNumber;
+        let mcn:number = tcn - lcn;
+        let wgd = null;
```

Comment:
always use `const` instead of `let`, unless you intend to update the value of the variable later.